src/components/content/content.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Person } from '../../types/Person';
+
+interface Props {
+  peopleFromServer: Person[];
+  setSelectedPerson: React.Dispatch<React.SetStateAction<null | Person>>;
+}
+
+const Content: React.FC<Props> = ({ peopleFromServer, setSelectedPerson }) => {
+  return (
+    <div className="dropdown-content">
+      {peopleFromServer.map(person => (
+        <div
+          className="dropdown-item"
+          data-cy="suggestion-item"
+          key={person.slug}
```

Comment:
Ensure that `person.slug` is unique for each `Person` object. If `slug` is not guaranteed to be unique, consider using a different unique identifier for the `key` prop to prevent potential rendering issues.